### PR TITLE
kv/mdbx: Ability to list tables from read-only tx, and enumerate DBs open in a process

### PR DESCRIPTION
**Pull request by AlexeyAkhunov**
This is for the diagnostics project

### CODE REVIEW

File: kv/mdbx/kv_mdbx.go [hunk 1]
```diff
@@ -368,6 +371,12 @@ func (opts MdbxOpts) Open() (kv.RwDB, error) {
 		}
 
 	}
+	if !opts.inMem {
```
Comment (AskAlexSharov) [inline kv/mdbx/kv_mdbx.go:374]: - maybe for in-mem case also can add/remove
- create helper funcs: PathDbMap, addDBPath, deleteDBPath

File: kv/mdbx/kv_mdbx.go [hunk 2]
```diff
@@ -443,7 +453,21 @@ func (db *MdbxKV) Close() {
...
+		pathDbMapLock.Lock()
+		defer pathDbMapLock.Unlock()
+		delete(pathDbMap, db.path)
+	}
+}
+
+func PathDbMap() map[string]kv.RoDB {
+	pathDbMapLock.Lock()
+	defer pathDbMapLock.Unlock()
+	result := map[string]kv.RoDB{}
```
Comment (AskAlexSharov) [inline kv/mdbx/kv_mdbx.go:466]: FYI: now can use `maps.Clone()`

**Review by AskAlexSharov — APPROVED** [review]

File: kv/kv_interface.go [hunk 3]
```diff
@@ -298,6 +298,7 @@ type StatelessRwTx interface {
 //   - ReadOnly transactions do not lock goroutine to thread, RwTx does
 type Tx interface {
 	StatelessReadTx
+	BucketMigratorRO
```
Comment (AskAlexSharov) [inline kv/kv_interface.go:301]: can add method ListBuckets() rigth to Tx interface, without creating BucketMigratorRO

Comment (AlexeyAkhunov) [reply kv/kv_interface.go:301]: I did that initially, but turned out `BucketMigrator` also needs it

File: kv/mdbx/kv_mdbx.go (hunk 1 above)
Comment (AlexeyAkhunov) [reply kv/mdbx/kv_mdbx.go]: done

File: kv/mdbx/kv_mdbx.go (hunk 2 above)
Comment (AlexeyAkhunov) [reply kv/mdbx/kv_mdbx.go]: done

**Pull request merged**